code/index.html

```diff
@@ -1,22 +1,180 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lexend+Giga&family=Noto+Sans+JP:wght@300&family=Oswald&family=Roboto:wght@300&display=swap"
+      rel="stylesheet"
+    />
   </head>
+  <header>
+    <img class="logo" src="./images/logo.svg" />
+    <nav>
+        <a class="navink" href="#">About</a>
+        <a class="navlink" href="#">Subscribe</a>
+        <a class="navlink" href="#">Contribute</a>
+        <a class="navlink" href="#">Enjoy</a>
+      </div>
+    </nav>
+  </header>
+
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
+    <!-- BIG NEWS SECTION -->
+    <div class="row">
+      <div class="bignews textbox">
+        <h1>We deliver great news daily!</h1>
+        <div class="button">EXPLORE</div>
+      </div>
+      <div class="bignews object-fit">
+        <a href="#">
+          <img src="./images/2.jpg" alt="broken image" />
+          <!-- <video class="bignewsvideo" autoplay>
+            <source src="./images/video1.mp4" type="video/mp4" />
+            Your browser does not support the video tag.
+          </video> -->
+        </a>
+      </div>
+    </div>
+    
+    <!-- FLEXBOX GALLERY-->
+    <section class="container">
+      <div class="message">
+        <h3>Get in the mood!</h3>
+      </div>
+<div class="gallery">
+        <a class="gallery-item" href="#"
+          ><img src="./images/1.jpg" alt="No image" /><span class="text-wrapper"
+            ><span class="title">It is just science, baby!</span
+            ><span class="explanation">
+              About asymmetrical roof party.</span
+            ></span
+          ></a
+        >
+        <a class="gallery-item" href="#"
+          ><img src="./images/2.jpg" allt="No image available" /><span
+            class="text-wrapper"
+            ><span class="title">It is just science, baby!</span
+            ><span class="explanation">
+              About asymmetrical roof party.</span
+            ></span
+          ></a
+        >
+        <a class="gallery-item" href="#"
+          ><img src="./images/1.jpg" alt="No image available" /><span
+            class="text-wrapper"
+            ><span class="title">It is just science, baby!</span
+            ><span class="explanation">
+              About asymmetrical roof party.</span
+            ></span
```

Comment:
Nice with the use of spans, the only thing I would advise you to change is the `<span>`-tag you're using that has the class="title". If something is a title or a subtitle it would be better to use a `<h[number]>`-tag. It's more semantically correct. The class="explantion"-span is great and used just as intended 👏🎖